modules/cli/src/main/java/org/apache/ignite/internal/cli/call/connect/DisconnectCall.java

```diff
@@ -34,9 +35,12 @@
 public class DisconnectCall implements Call<EmptyCallInput, String> {
     @Inject
     private final Session session;
+    @Inject
```

Comment:
Micronaut injects the constructor arguments automatically so there's no need to annotate the fields.